src/components/LandingSection.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from "react";
+import {
+  Grid,
+  Row,
+  Column,
+  TextInput,
+  Button,
+  ToastNotification,
+} from "carbon-components-react";
+
+const LandingSection = () => {
+  const [error, setError] = useState("");
+  const [id, setId] = useState("");
+  const [showToaster, setShowToaster] = useState(false);
+
+  const handleBtnClose = () => {
+    setShowToaster(false);
+  };
+
+  const checkInput = (val) => {
+    setError("");
+    setId(val);
+  };
+
+  function isAlphanumeric(str) {
+    return /^[a-zA-Z0-9]+$/.test(str);
+  }
+
+  const submitID = (e) => {
+    e.preventDefault();
+    if (!isAlphanumeric(id)) {
+      setError("Only alphanumeric input is allowed");
+    } else {
+      setError("");
+      setId("");
+      setShowToaster(true);
+    }
+  };
+
+  return (
+    <Grid data-testid="landing-section-container" className="hero__container">
+      <Row className="hero__content">
+        <Column className="hero__text">
+          <div className="hero__section">
+            <h1>
+              Welcome to <span>Patient Portal</span> App
```

Comment:
We will need a story in JIRA to externalise all user facing strings into a messages bundle for translation. Patient facing portal will likely need ability for people to select their local language. Can you please create a new story in JIRA for this?

Reply:
Okay sure, will create a Jira story for this. Will we be using i18n for this purpose?